.github/workflows/build_all.yml

```
@@ -555,6 +555,8 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           pip install Pyrebase
```

There is also the following alternative suggested fix, which looks more promising:

```suggestion
          pip3 install pyrebase4
```